src/app/pages/virtualization/components/all-instances/all-instances.component.html

```diff
@@ -1,22 +1,24 @@
-<ix-page-header>
-  <ix-all-instances-header></ix-all-instances-header>
-</ix-page-header>
+<ix-master-detail-view
+  #masterDetailView="masterDetailViewContext"
+>
+  <ix-page-header header>
```

Comment:
I don't think page header needs to be part of this directive.

Reply:
ah, yeah, I got you.
Thanks, I'll fix that.